meta/correction.go

```diff
@@ -37,26 +38,34 @@ func (set *Set) Corrections(coll Collection) []Correction {
 					continue
 				}
 
-				for _, sensor := range set.SensorCalibrationCorrections(coll) {
-					span, ok := span.Extent(sensor.Span)
+				for _, telemetry := range set.TelemetryCorrections(coll) {
+					span, ok := span.Extent(telemetry.Span)
 					if !ok {
 						continue
 					}
 
-					for _, datalogger := range set.DataloggerCalibrationCorrections(coll) {
-						span, ok := span.Extent(datalogger.Span)
+					for _, sensor := range set.SensorCalibrationCorrections(coll) {
+						span, ok := span.Extent(sensor.Span)
 						if !ok {
 							continue
 						}
 
-						corrections = append(corrections, Correction{
-							Span:                  span,
-							Polarity:              polarity.Polarity,
-							Gain:                  gain.Gain,
-							Preamp:                preamp.Preamp,
-							SensorCalibration:     sensor.SensorCalibration,
-							DataloggerCalibration: datalogger.DataloggerCalibration,
-						})
+						for _, datalogger := range set.DataloggerCalibrationCorrections(coll) {
+							span, ok := span.Extent(datalogger.Span)
+							if !ok {
+								continue
+							}
+
+							corrections = append(corrections, Correction{
+								Span:                  span,
+								Polarity:              polarity.Polarity,
+								Gain:                  gain.Gain,
```

Comment:
does this need to change to ScaleFactor as well? (there might be other places in the codes, only commenting here)

Reply:
No, that one's okay. It's linking to the full Gain list (i.e. the gains.csv input), there will be a gain.Gain.ScaleFactor.

Reply:
ah gotcha!! THANKS!!!